### DISCUSSION TITLE
Quotes arround tooltip when provided as stringi n Flet 0.24.0

### DISCUSSION THREAD

**Discussion by mvwilpe** [flet-dev/flet#3913] [General]
IconButon double quotes strings around tooltip.
When provided as string tooltip="This is the tooltip" this is displayed as "This is the tooltip"
When provided as Tooltip object: Tooltip(message="Another tooltip")  this is displayed as {"message": "Another tooltip"}

It seems that - at least for the IconButton - the the json converted value of tooltip is displayed regardless whether is provided as string or as Tooltip object.
Textbutton does not have this problem (I've not tried other type of controls)

**Comment by ndonkoHenri** [comment]
https://github.com/flet-dev/flet/issues/3920
